k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant2/nfdiv-cron-remind-applicant2.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-remind-applicant2
+spec:
+  releaseName: nfdiv-cron-remind-applicant2
+  chart:
+    git: git@github.com:hmcts/nfdiv-cron
+    ref: 0.0.7
+    path: nfdiv-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-00fe383-20210826060439  #{"$imagepolicy": "flux-system:nfdiv-case-api"}
```

Comment:
Not the latest image, unless you want to imageAutomation to update this when merged?

Reply:
updated to latest image but we will also rely on image automation